src/api/public/media/media.controller.ts

```diff
@@ -51,6 +50,7 @@ import {
   successfullyDeletedDescription,
   unauthorizedDescription,
 } from '../../utils/descriptions';
+import { ApiImplicitFile } from '@nestjs/swagger/dist/decorators/api-implicit-file.decorator';
```

Comment:
This import is weird.
According to https://github.com/nestjs/swagger/issues/629, this decorator is not in `@nestjs/swagger`v4 anymore. I also cand find it in the[current repo]( https://github.com/nestjs/swagger/search?q=ApiImplicitFile&type=code)

Maybe use the `ApiBody` decorator like explained in https://github.com/nestjs/swagger/issues/417#issuecomment-562869578?

Reply:
Changed to suggested `@ApiBody`